### DISCUSSION TITLE
如何将图层中的文字描述手动全部隐藏，展示进行切换

### DISCUSSION THREAD

**Discussion by 191421043** [frangoteam/FUXA#835] [Q&A]
通过拖拽画出来的控件，附加文字描述，怎么把所有的文字全部隐藏或者显示，进行随意切换

**Comment by jbryan189** [comment]
不确定我是否清楚需求，但我将文本和显示的设备分组并复制粘贴项目。然后编辑参考文献就完成了。

Not sure I'm clear about the needs, but I group the text and displayed device and copy paste item. Then edit the references and it's done.
